packages/block-editor/src/components/spacing-sizes-control/input-controls/spacing-input-control.js

```diff
@@ -33,7 +33,7 @@ import {
 } from '../utils';
 import { unlock } from '../../../lock-unlock';
 
-const { CustomSelectControlV2Legacy: CustomSelectControl } = unlock(
+const { CustomSelectControlV2: CustomSelectControl } = unlock(
```

Comment:
@ciampo Looking at the description in https://github.com/WordPress/gutenberg/pull/63190, should this be using the V1 implementation?

Reply:
Thanks for looking into this @ramonjd! I was just wondering that from looking at this PR, whether we should be using `CustomSelectControl` as in https://github.com/WordPress/gutenberg/pull/63258 instead of adding `CustomSelectControlV2` to the private APIs at this stage?

I imagine @ciampo or @mirka will know which is preferred at this stage 🙂

Reply:
This problem appears to be solved if we import the `CustomSelectControl` component directly from the `@wordpress/components` package without using unlock. However, I'm not confident about it yet, so I'll wait for feedback from the component team 😅

Reply:
I think we're asking the same question 😄 

Using v1 is even easier. 

I'll add a commit to save time later. We can swap out one for the other depending on what the components avengers say. 🦸🏻

Reply:
v1 in [629a94a](https://github.com/WordPress/gutenberg/pull/64284/commits/629a94a4b428d9d95bca7e9c1ca3aa45d932c111)

Reply:
Thank you for sorting this out 😱 What you did here is correct!